site/public/css/rainbow-grades.css

```diff
@@ -1,3 +1,11 @@
 #rainbow-grades > table {
     color: black;
 }
+
+#rainbow-grades-header > .btn-wrapper {
+    margin: auto;
+}
+
+#rainbow-grades-header > p {
+    float: right;
+}
```

Comment:
Targeting elements by raw selectors like `p` is discouraged.  Please use a class or ID instead.  You can also probably find an existing CSS selector to do this.